pkg/controller/elasticsearch/observer/state.go

```diff
@@ -28,19 +27,18 @@ type State struct {
 // RetrieveState returns the current Elasticsearch cluster state
 func RetrieveState(ctx context.Context, cluster types.NamespacedName, esClient esclient.Client) State {
 	// retrieve both cluster state and health in parallel
-	clusterStateChan := make(chan *client.ClusterState)
+	infoChan := make(chan *client.Info)
 	healthChan := make(chan *client.Health)
 	licenseChan := make(chan *client.License)
 
 	go func() {
-		clusterState, err := esClient.GetClusterState(ctx)
+		info, err := esClient.GetClusterInfo(ctx)
 		if err != nil {
-			// This is expected to happen from time to time
-			log.V(1).Info("Unable to retrieve cluster state", "error", err, "namespace", cluster.Namespace, "es_name", cluster.Name)
-			clusterStateChan <- nil
+			log.V(1).Info("Unable to retrieve cluster info", "error", err, "namespace", cluster.Namespace, "es_name", cluster.Name)
```

Comment:
Is this still expected since we changed the endpoint or should we switch this to an error?